scripts/lib-franklin.js

```diff
@@ -144,6 +144,11 @@ export function decorateIcons(element = document) {
       } else {
         span.innerHTML = iconHTML;
       }
+      const a = span.closest('a');
+      if (a && !a.title) {
+        const iconName = span.classList[1];
+        a.title = iconName.substring(5) || `${a.href}`;
```

Comment:
Will  `a.href` do the same thing as ``${a.href}``?

Reply:
Yes I believe it will.